scadalts-ui/src/store/modernWatchList/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author rjajko@softq.pl
+ */
+const modernWatchList = {
+    state: {
+        activeWatchList: undefined,
+        activeUser: undefined,
```

Comment:
We do not define values as undefined as you want to initiate them as null or not.